block/Makefile

```diff
@@ -36,3 +36,6 @@ obj-$(CONFIG_BLK_DEBUG_FS)	+= blk-mq-debugfs.o
 obj-$(CONFIG_BLK_DEBUG_FS_ZONED)+= blk-mq-debugfs-zoned.o
```

Comment:
This 'move io filter' commit could get folded into the commit that introduced these files.